sql/catalyst/src/main/antlr4/org/apache/spark/sql/catalyst/parser/SqlBase.g4

```diff
@@ -172,6 +172,10 @@ unsupportedHiveNativeCommands
     | kw1=ROLLBACK
     | kw1=DFS
     | kw1=DELETE kw2=FROM
+    | kw1=RELOAD kw2=FUNCTION
+    | kw1=DELETE kw2=(FILE | FILES | JAR | JARS | ARCHIVE | ARCHIVES)
```

Comment:
Could you perhaps do this without adding the `FILE`, `FILES`, `JAR`, `JARS`, `ARCHIVE` & `ARCHIVES` keywords? This makes it consistent with the `ADD <resource>` command.